index.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="KO">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Linkbrary</title>
+    <link rel="stylesheet" href="/style.css">
+</head>
+<body>
+    <nav class="navigation-bar">
+        <a href="/"><img src="./image/logo.png" alt="linkbrary logo" width="133px" height="24px"></a>
+        <span class="login-button"><a href="/signin.html">로그인</a></span>
```

Comment:
질문 주신 부분인데요~ 
요렇게 해주시면 로그인 버튼이 잘 동작할 것 같네요~

```suggestion
        <a class="login-button" href="/signin.html">로그인</a>
```